docs/capabilities/architecture-quality.md

```diff
@@ -0,0 +1,60 @@
+# Architecture Quality
+
+Software engineering is a socio-technical activity. Software is built by teams of people, working together to produce a joint product. High performers in socio-technical architecture quality show faster issue resolution times. Sigrid measures Architecture Quality Model to quantify these aspects of socio-technical software architecture, covering both technical and social aspects and then compares the results against [SIG’s benchmark](../getting-started/approach.md). 
+
+<iframe width="560" height="315" src="https://www.youtube.com/embed/BejJ9sHeUOQ" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
+
+When people talk about "legacy" or "monoliths", they often mean much more than just outdated technology. Rather, it refers to any pre-existing software solution that has become too fragile for changes to be timely, predictable, and reliable, usually due to poor architecture, or team knowledge loss. 
+
+Software architecture can seem abstract, but it's ultimately how the structure of your software facilitates the people working on it. The code will not care that it's highly coupled, but the people working on that code certainly do care about having dependencies on other people and teams. Sigrid can help you to both *communicate* the as-is architecture as it's actually implemented, and also help to find opportunities on how it can be improved. 
+
+## Architecture Quality overview
+
+The overview page provides a summary of your system's current architecture, relative to the other systems in SIG's benchmark. The structure of this page is comparable to Sigrid's [maintainability summary](system-maintainability.md): you have an overall rating, which is then split into a number of underlying system properties and associated ratings. Hovering your cursor over the system properties on the right provides you with a short description of each metric.
+
+<img src="../images/aq-summary.png" width="600" />
+
+This page provides you with a summary, but you may decide that further investigation into the actual architecture is needed. You can use the "explore architecture" button to navigate to Sigrid's architecture explorer page. Clicking on one of the system properties on the right will *also* take you to the architecture explorer page, but will then immediately jump to the system property you just clicked on.
+
+## Exploring your system's as-is architecture
```

Comment:
Maybe we can say "your system's current architecture" to indicate that this is actually the "current" architecture? It would link well with the first paragraph immediately below and it makes it maybe more obvious to the readers.

Reply:
Will do. We do actually want to have change tracking in both the summary and explorer pages, but at the moment they indeed only cover the current architecture.